src/eager/raw_ops.rs

```diff
@@ -0,0 +1,82 @@
+/// Tentative implementation of raw_ops for unit testing.
```

Comment:
If this is only for unit testing, then the module should be hidden behind a `#[cfg(test)]`, and the name should somehow reflect that it contains test helpers.

Reply:
I've moved them down one level to make it more obvious that they are helper functions for Op unit tests. I will also put raw_ops, which will be added later, in this position.